Source/Scene/Scene.js

```diff
@@ -2794,13 +2916,50 @@ define([
             }
             scene._cameraMovedTime = getTimestamp();
             Camera.clone(camera, scene._cameraClone);
-        } else if (scene._cameraStartFired && getTimestamp() - scene._cameraMovedTime > scene.cameraEventWaitTime) {
+
+            return true;
```

Comment:
Relying on this function to determine whether or not the camera moved is going to be problematic. See #4753

Reply:
Would you recommend using the `camera.changed` event instead? I can also update the `moveStart`/`moveEnd` events to fire according to that rather than using `cameraEqual`.

Reply:
Nope, `camera.changed` is broken for small camera changes as well.  We'll pretty much have to fix #4753 in order to determine if the camera changed.

Reply:
@bagnell, do you know if there is a fix or a workaround we could use here?

Reply:
We would need to fix #4753, but you can change the epsilon test to an equality test in `cameraEqual`.